gui/locales/fr.json

```diff
@@ -84,25 +84,25 @@
   "Help Our community grows everyday and will be happy to give you an helping hand in one of these media:": "Notre communauté grandit tous les jours et sera ravie de vous aider sur l'un de ses médias :",
   "Help Forum": "Forum",
   "Help IRC": "IRC",
-  "Help Github": "Github",
+  "Help Github": "GitHub",
   "Help Official Support": "Support officiel",
   "Help Hello Cozy,": "Bonjour Cozy,",
-  "Help I like a lot what you do, but I have an issue:": "J'aime beaucoup ce que vous faites, mais j'ai un problème :",
+  "Help I like a lot what you do, but I have an issue:": "J’aime beaucoup ce que vous faites, mais j’ai un problème :",
   "Help [ The more you can say about the issue, the better: do you have many files? Are they big? Is your cozy up-to-date? ]": "[ Donnez nous un maximum de détails : avez-vous beaucoup de fichiers ? Sont-ils gros ? Votre cozy est-il à jour ? ]",
   "Help Take care!": "À bientôt !",
-  "Help Your mail has been sent. We will try to respond to it really soon!": "Votre courriel a été envoyé. Nous essayerons d'y répondre très rapidement !",
-  "Help An error occured while sending your email": "Une erreur est survenue pendant l'envoi de votre courriel. Veuillez contacter le support.",
-  "Help You can send us feedback, report bugs and ask for assistance.": "Vous pouvez nous envoyer vos retours, nous remonter des bugs et demander de l'assistance.",
-  "Help We will get back to you as soon as possible.": "Nous allons revenir vers vous aussi vite que possible.",
-  "Help Multi Computer": "Si vous utilisez plusieurs ordinateurs, envoyer également un message depuis les autres ordinateurs nous aidera à identifier l'erreur.",
+  "Help Your mail has been sent. We will try to respond to it really soon!": "Votre email a été envoyé. Nous essayerons d’y répondre très rapidement !",
+  "Help An error occured while sending your email": "An error occured while sending your email. Please contact the support.",
+  "Help You can send us feedback, report bugs and ask for assistance.": "Vous pouvez nous envoyer vos retours, nous remonter des bugs et demander de l’aide.",
+  "Help We will get back to you as soon as possible.": "Nous reviendrons vers vous aussi vite que possible.",
+  "Help Multi Computer": "If you use several computers with this cozy, sending the log from the other ones as well will help us pinpoint the issue.",
```

Comment:
English translation in fr locale file ?

Reply:
IDK, just did `tx push -s` then `tx pull` then commit this commit https://github.com/cozy-labs/cozy-desktop/pull/1222/commits/c4e45657c95eaa9a8be719c55d8ab20d10a5a0ff

My need was to treat https://github.com/cozy-labs/cozy-desktop/pull/1222/commits/eb41b42ca9d3dd7da1fdb0a2ed0fbe5794b2d9fb

When browsing transifex website I saw 2 issues and I created https://github.com/cozy-labs/cozy-desktop/pull/1222/commits/459367f1b45de67216a9ece3c44ef5f0386946fd